src/languages/es.js

```diff
@@ -680,6 +680,17 @@ export default {
             },
         },
     },
+    colorThemePage: {
+        colorTheme: 'Tema de color',
+        colorThemes: {
+            dark: {
+                label: 'Oscura',
```

Comment:
note to self: get these checked off in slack

Reply:
@grgia we'll still need to do that

Reply:
on it!

Reply:
for my later ref - https://expensify.slack.com/archives/C21FRDWCV/p1688552985290959

Reply:
Dark -> `Oscuro`
Light -> `Claro`
Use Device Settings -> `Utiliza los ajustes del dispositivo`
Choose a theme below, or sync with your device settings.  -> `Elige un tema a continuación o sincronízalo con los ajustes de tu dispositivo.`